db/migrate/20230929103257_add_information_about_multiple_session.rb

```diff
@@ -0,0 +1,16 @@
+class AddInformationAboutMultipleSession < ActiveRecord::Migration[6.1]
+  def change
+    add_column :user_sessions, :multiple_fill, :boolean
+
+    execute(<<-SQL.squish)
+        UPDATE user_sessions
+        SET multiple_fill = (
+          SELECT sessions.multiple_fill
+          FROM sessions
+          WHERE sessions.id = user_sessions.session_id
+        );
+    SQL
+
```

Comment:
add require for 'multiple_flag' after this operation so we will be sure that we don't miss anything